examples/simple_reverse.coffee

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env coffee
+# This is a simple example of how to use the slack-client module in CoffeeScript. It creates a
+# bot that responds to all messages in all channels it is in with a reversed
+# string of the text received.
+#
+# To run, copy your token below, then:
+#  npm install
+#   cd examples
+#   coffee simple_reverse.coffeescript
```

Comment:
Wrong extension.  Also probably simpler to compile this to JS and use `node` to run it (these instructions as is would also require having coffee-script globally installed).